src/layout/Summary2/config.tsx

```diff
@@ -0,0 +1,67 @@
+import { CG, Variant } from 'src/codegen/CG';
+import { CHECKBOX_SUMMARY_PROPS } from 'src/layout/Checkboxes/config';
+import { CompCategory } from 'src/layout/common';
+import { DROPDOWN_SUMMARY_PROPS } from 'src/layout/Dropdown/config';
+import { INPUT_SUMMARY_PROPS } from 'src/layout/Input/config';
+import { MULTIPLE_SELECT_SUMMARY_PROPS } from 'src/layout/MultipleSelect/config';
+import { RADIO_SUMMARY_PROPS } from 'src/layout/RadioButtons/config';
+export const Config = new CG.component({
+  category: CompCategory.Container,
+  rendersWithLabel: false,
+  capabilities: {
+    renderInTable: false,
+    renderInButtonGroup: false,
+    renderInAccordion: false,
+    renderInAccordionGroup: false,
+    renderInCards: false,
+    renderInCardsMedia: false,
+  },
+})
+  .addProperty(
+    new CG.prop(
+      'children',
+      new CG.arr(new CG.str())
+        .optional()
+        .setTitle('Children')
+        .setDescription('Array of component IDs that should be displayed in the summary'),
+    ).onlyIn(Variant.External),
+  )
+  .addProperty(new CG.prop('childComponents', new CG.arr(CG.layoutNode)).onlyIn(Variant.Internal))
+  .addProperty(
+    new CG.prop(
+      'children',
+      new CG.arr(new CG.str())
+        .optional()
+        .setTitle('Children')
+        .setDescription('Array of component IDs that should be displayed in the summary'),
```

Comment:
I would love to get a clearer description about how this works as opposed to `whatToRender`.. Also, if it's meant to be used together with `whatToRender` - how does that play out? This is what I would expect to see in a description, so I don't have to try it out or just guess when configuring this component.

Reply:
This is leftovers, we only use `whatToRender` :)